data-raw/process_raw_data.R

```diff
@@ -0,0 +1,6 @@
+sierra_leone_ebola_outbreak_data <- readxl::read_xlsx(
```

Comment:
```suggestion
sierra_leone_ebola_outbreak_data <-
	readxl::read_xlsx(
```

Reply:
What's the style guide here?

Reply:
I mean this isn't ideal but is what `lintr` will suggest with these very long names. I agree with your name change so can resolve this.